src/Sulu/Bundle/ContactBundle/Contact/AccountManager.php

```diff
@@ -168,6 +169,26 @@ public function getById($id, $locale)
     }
 
     /**
+     * Returns all media for an account with a given id.
+     *
+     * @param $id
+     * @param $locale
+     *
+     * @return Media[]
+     *
+     * @throws EntityNotFoundException
+     */
+    public function getMediaById($id, $locale)
```

Comment:
Why aren't you just using `getById` and  call `getMedias()` in the controller? Same question for the contacts.